src/modules/dashboard/lists/ListHeader.tsx

```diff
@@ -13,15 +13,11 @@ interface ListHeaderProps {
 export const ListHeader = ({ head }: ListHeaderProps) => {
   return (
     <ListHeaderWrapper>
-      <ListColumn maxWidth={160} isRow>
-        <ListHeaderTitle>
-          <Trans>Assets</Trans>
-        </ListHeaderTitle>
-      </ListColumn>
-
       {head.map((title, i) => (
-        <ListColumn overFlow={'visible'} key={i}>
-          <ListHeaderTitle>{title}</ListHeaderTitle>
+        <ListColumn overFlow={'visible'} key={i} isRow={i === 0}>
+          <ListHeaderTitle>
+            <Trans>{title}</Trans>
```

Comment:
Could you remove the <Trans> tag here, it won't have an affect since {title} is a ReactNode and the title values are already wrapped in <Trans> in BorrowAssetsList, SupplyAssetsList, etc.

Other than that looks perfect, nice work!